index.js

```diff
@@ -160,3 +162,73 @@ module.exports.hexToUint8 = function (hex/* : string */ = '') {
   }
   return arr
 }
+
+/**
+ * Utilities for converting keys to passphrases using bip39 or niceware
+ */
+module.exports.passphrase = {
+  /* @exports passphrase */
+  /**
+   * Converts bytes to passphrase using bip39 (default) or niceware
+   * @method
+   * @param {Uint8Array|Buffer|string} bytes Uint8Array / Buffer / hex to convert
+   * @param {boolean=} useNiceware Whether to use Niceware; defaults to false
+   * @returns {string}
+   */
+  fromBytesOrHex: function (bytes/* : Uint8Array | string */, useNiceware/* : boolean */ = false) {
+    if (useNiceware) {
+      if (typeof bytes === 'string') {
+        bytes = module.exports.hexToUint8(bytes)
+      }
+      return niceware.bytesToPassphrase(Buffer.from(bytes)).join(' ')
+    } else {
+      if (typeof bytes !== 'string') {
+        bytes = module.exports.uint8ToHex(bytes)
+      }
+      return bip39.entropyToMnemonic(bytes)
+    }
+  },
+
+  /**
+   * Converts a 32-byte passphrase to uint8array bytes. Infers whether the
+   * passphrase is bip39 or niceware based on length.
+   * @method
+   * @param {string} passphrase bip39/niceware phrase to convert
+   * @returns {Uint8Array}
+   */
+  toBytes32: function (passphrase/* : string */) {
+    passphrase = passphrase.trim().replace(/\s+/gi, ' ')
+    const words = passphrase.split(' ')
+    if (words.length === 16) {
```

Comment:
You may capture this as a constant (would eliminate the need for the `// niceware` comment below, since it could be like `NICEWARE_COUNT`)